### PR TITLE
Apply official clang format style

### PR DESCRIPTION
**Pull request by ofrobots**
This is dependent on #141 and #142.

### CODE REVIEW

File: src/transporters.ts [hunk 1]
```diff
@@ -69,17 +72,17 @@ export default class DefaultTransporter {
       // responds without proper content-type.
       try {
         body = JSON.parse(body);
-      } catch (err) { /* no op */ }
+      } catch (err) { /* no op */
```
Comment (JustinBeckwith) [inline src/transporters.ts]: really?

File: src/transporters.ts [hunk 2]
```diff
@@ -69,17 +72,17 @@ export default class DefaultTransporter {
...
         if (typeof body.error === 'string') {
           err = new Error(body.error);
           err.code = res.statusCode;
 
         } else if (Array.isArray(body.error.errors)) {
-          err = new Error(body.error.errors.map(
-                          (err2) => err2.message
-                        ).join('\n'));
+          err = new Error(
+              body.error.errors.map((err2) => err2.message).join('\n'));
```
Comment (JustinBeckwith) [inline src/transporters.ts]: Do we have an opinion in the style guide on using no parens with single argument lambdas?

File: src/transporters.ts (hunk 1 above)
Comment (ofrobots) [inline src/transporters.ts]: I have added a manual fixup for this. Please open bugs against clang-format upstream the style-guide.

File: src/transporters.ts (hunk 2 above)
Comment (ofrobots) [inline src/transporters.ts]: The style guide allows it: https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions. Clang-format will keep existing style though when formatting.

I have manually removed the parens.

**Comment by ofrobots** [comment]
This is rebased + I have addressed some nits. I've also added check-format to the `npm test`. If the CI is green, I will land tomorrow.

**Pull request merged**